client/examples/http_client.rs

```diff
@@ -108,6 +114,61 @@ struct Ping {
     response: oneshot::Sender<Result<String, String>>,
 }
 
+struct DisplayableNode(Node);
+
+impl fmt::Display for DisplayableNode {
```

Comment:
MAybe it is worth implementing it directly on `Node`

Reply:
It would't allow me to do it here and it's rather specific to what we have in the example so I didn't want to modify the original Node implementation